### ISSUE TITLE
docs-presubmit failing

### ISSUE THREAD

**Issue by larkee** [googleapis/python-spanner#128]
```
[18:13:16][ERROR] Failed to get build config
com.google.devtools.kokoro.config.ConfigException: Couldn't find build configuration file docs-presubmit.cfg or docs-presubmit.gcl under /tmp/workspace/workspace/cloud-devrel/client-libraries/python/googleapis/python-spanner/docs/docs-presubmit/src/github/python-spanner/.kokoro/docs.
	at com.google.devtools.kokoro.config.BuildConfigReader.lambda$read$2(BuildConfigReader.java:54)
	at java.util.Optional.orElseThrow(Optional.java:290)
	at com.google.devtools.kokoro.config.BuildConfigReader.read(BuildConfigReader.java:51)
	at com.google.devtools.kokoro.jenkins.plugin.kokorojob.store.NodeBuildConfigReader.invoke(NodeBuildConfigReader.java:39)
	at com.google.devtools.kokoro.jenkins.plugin.kokorojob.store.NodeBuildConfigReader.invoke(NodeBuildConfigReader.java:13)
	at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2731)
	at hudson.remoting.UserRequest.perform(UserRequest.java:153)
	at hudson.remoting.UserRequest.perform(UserRequest.java:50)
	at hudson.remoting.Request$2.run(Request.java:336)
	at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.jenkinsci.remoting.kokoro.RpcSlaveEngine$1$1.run(RpcSlaveEngine.java:107)
	at java.lang.Thread.run(Thread.java:748)
	at ......remote call to gcp_ubuntu-prod-yoshi-ubuntu-ir-819542672(Native Method)
	at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1537)
	at hudson.remoting.UserResponse.retrieve(UserRequest.java:253)
	at hudson.remoting.Channel.call(Channel.java:822)
	at hudson.FilePath.act(FilePath.java:985)
	at hudson.FilePath.act(FilePath.java:974)
	at com.google.devtools.kokoro.jenkins.plugin.kokorojob.store.ConfigStore.getKokoroBuildConfig(ConfigStore.java:102)
	at com.google.devtools.kokoro.jenkins.plugin.pipeline.KokoroFlowExecution.getBuildConfig(KokoroFlowExecution.java:661)
	at com.google.devtools.kokoro.jenkins.plugin.pipeline.KokoroFlowExecution.addPostScmSteps(KokoroFlowExecution.java:608)
	at com.google.devtools.kokoro.jenkins.plugin.pipeline.KokoroScmStepContext.onSuccess(KokoroScmStepContext.java:25)
	at org.jenkinsci.plugins.workflow.steps.AbstractSynchronousNonBlockingStepExecution$1.run(AbstractSynchronousNonBlockingStepExecution.java:44)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by hengfengli** [comment]
@larkee I think this issue has already been fixed, right?

**Comment by larkee** [comment]
Correct. This was fixed in #126
